config/locales/en.yml

```diff
@@ -56,9 +56,9 @@ en:
       title: "Provide patient with COVID Shield code"
       subheading: "Instruct the patient to:"
       step1: "Open the COVID Shield app by tapping on the icon."
-      step2: "Tap the white section at the bottom of the screen."
+      step2: "Tap the <strong>COVID Shield status section</strong> at the bottom of the screen."
```

Comment:
Let's add context for screen reader users.

Change this…
```html
<strong>COVID Shield status section</strong>
```

to…
```html
<strong>COVID Shield status <span aria-hidden="true">section</span><span class="visually-hidden">button</span></strong>
```

Sighted users will read `section` and screen reader users will hear `button`; better for context.